test/counts.jl

```diff
@@ -85,6 +85,10 @@ pm = proportionmap(x)
 @test pm["b"] ≈ (1/3)
 @test pm["c"] ≈ (1/6)
 
+xx = vcat(repeat([1];inner = 100), rand(2:1000, 1_000_000))[sample(1:1_000_100, 1_000_100, replace=false)]
```

Comment:
Better use a simpler pattern, and check that the result is correct. For example, call `repeat([1, 1, 3, 6], outer=100_000)`, and hardcode the result for the check. Also, a comment saying what is being tested would be helpful given that it's not obvious.